pkpdapp/pkpdapp/models/biomarker_type.py

```diff
@@ -9,18 +9,20 @@
 
 class BiomarkerType(models.Model):
     """
-    A biomarker type, for example "concentration in mg". Each
-    :model:`pkpdapp.Biomarker` is assigned on of these types.
+    A generic (not associated with a particular dataset) biomarker type, for
+    example "concentration in mg", or "volume in cm^3".
     """
 
     UNIT_CHOICES = [
-        ('mg', 'g'),
+        ('mg', 'mg'),
```

Comment:
Wouldn't this make it possible to create a volume that had units g? Is it worth separating out BiomarkerMeasurements from BiomarkerUnits where there's a one to many mapping from the former to the latter? But that those mappings are hard-coded?

Reply:
Yea. I thinking that it might be better to include "units" as a field in BiomarkerMap, and get rid of BiomarkerType entirely, what do you think?